salt/orchestrate/services/elasticsearch.sls

```diff
@@ -1,14 +1,16 @@
 {% set env_settings = salt.cp.get_file_str("salt://environment_settings.yml")|load_yaml %}
 {% set ENVIRONMENT = salt.environ.get('ENVIRONMENT', 'rc-apps') %}
 {% set env_data = env_settings.environments[ENVIRONMENT] %}
+{% set purpose_data = env_data.purposes.elasticsearch|default({}) %}
 {% set VPC_NAME = env_data.vpc_name %}
-{% set INSTANCE_COUNT = salt.environ.get('INSTANCE_COUNT', 3) %}
+{% set INSTANCE_COUNT = salt.environ.get('INSTANCE_COUNT', purpose_data.num_instances|default(3)) %}
```

Comment:
Instead of using the Jinja default filter I would just do `purpose_data.get('num_instances', 3)`

Reply:
✅